### DISCUSSION TITLE
{"error":"file_exists","reason":"The database could not be created, the file already exists."}

### DISCUSSION THREAD

**Discussion by crohit7** [Budibase/budibase#7061] [Help] [answered]
I am getting error like  {"error":"file_exists","reason":"The database could not be created, the file already exists."}
while I run the command docker-compose up

I am hosting budibase on Azure Ubuntu 20.04 LTS Vm

Below I attched screenshot

Please help me!

![image](https://user-images.githubusercontent.com/37010744/182602932-d58b325f-c570-4ec5-8089-5d8b8f012cda.png)

**Comment by mike12345567** [comment, marked as answer]
Hi @crohit7 - you can ignore this error, its just our system confirming some databases are available at startup, if it doesn't error then it will create them. We should swallow this error to reduce confusion, I'll take a note of that.
